src/phpunit-command.js

```diff
@@ -91,7 +100,7 @@ module.exports = class PhpUnitCommand {
 
         while (line > 0) {
             const lineText = vscode.window.activeTextEditor.document.lineAt(line).text;
-            const match = lineText.match(/^\s*(?:public|private|protected)?\s*function\s*(\w+)\s*\(.*$/);
+            const match = vscode.workspace.getConfiguration('better-phpunit').get('usePest') ? lineText.match(/^\s*it\(([^,)]+)/m) : lineText.match(/^\s*(?:public|private|protected)?\s*function\s*(\w+)\s*\(.*$/);
```

Comment:
A test can be in one of the following 3 conditions:

```php
it('foo', function () {
    $this->get('/')->assertSee('Laravel');
});

test('foo', function () {
    $this->get('/')->assertSee('Laravel');
});

it('foo')->get('/')->assertSee('Laravel');

test('foo')->get('/')->assertSee('Laravel');

get('/')->assertSee('Laravel');
```

All of those lines are valid single tests.

Reply:
It does not (will update soon)

Reply:
This should handle all three methods

```suggestion
            const match = vscode.workspace.getConfiguration('better-phpunit').get('usePest') ? lineText.match(/^\s*(?:it|test|get)\(([^,)]+)/m) : lineText.match(/^\s*(?:public|private|protected)?\s*function\s*(\w+)\s*\(.*$/);
```

Reply:
I feel like for the last one (`get()`), it is not going to be feasible to handle all of them. Because we support `post('/')`, `postJson('/')`, and a load of other ones in the Laravel plugin.

https://github.com/pestphp/pest-plugin-laravel/blob/master/src/Http.php

I guess this would be possible if we checked for an import of `Pest\Laravel\{get,post,etc}` and matched it but that just complicates the process. Maybe it's better to just support `test`/`it` for now?